raccoon.log

```diff
@@ -0,0 +1,1890 @@
+[21-01-28 23:05:28] (DEBUG) Starting command: CS\Raccoon\Commands\AwsSshCommand, Input: 'aws:ssh'  
```

Comment:
> also if you could keep new line at the end of the files it would be nice

But where?
AFAIK I've got this enabled in my IDE as well as stripping trailing whitespace.